applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -726,19 +964,33 @@ namespace Kratos
         const bool CalculateResidualVectorFlag
         )
     {
-       KRATOS_ERROR << "You are calle to the CalculateAll from the base class for solid elements" << std::endl; 
+       KRATOS_ERROR << "You have called to the CalculateAll from the base class for solid elements" << std::endl; 
     }
     
     //***********************************************************************
     //***********************************************************************
 
     double BaseSolidElement::GetIntegrationWeight(
         const GeometryType::IntegrationPointsArrayType& ThisIntegrationMethod,
-        const unsigned int PointNumber,
+        const unsigned int point_number,
         const double detJ
         )
     {
-        return ThisIntegrationMethod[PointNumber].Weight() * detJ;
+        return ThisIntegrationMethod[point_number].Weight() * detJ;
+    }
+    
+    //************************************************************************************
+    //************************************************************************************
+    
+    void BaseSolidElement::UpdateKinematics(
```

Comment:
I am just saying look to the official current code in the master. The stress and strain is calculated manually. We are suppose to use the CL. And the code in CalculateAll is basically the same in small displacement and total lagrangian, and with this the code is simplified